firmware/config/boards/kinetis/prepend.txt

```diff
@@ -0,0 +1 @@
+#define ts_show_trigger_comparator true
```

Comment:
@rusefi @andreika-git it seems like this doesn't work, is there some trick to get the tool to pick this up?

Reply:
Try putting it inside firmware/config/boards/kinetis/config/rusefi_config_kinetis.txt

Reply:
Or firmware/config/boards/kinetis/config/tunerstudio/kinetis_prefix.txt

Reply:
The former suggestion seems to work - thanks!